scripts/setup-data-dir.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -e
+set -x
```

Comment:
I am pondering about `set -x`... if packagers use `scripts/setup-data-dir.sh`, it will increase the output of `dnf install` unnecessarily.